frontend/test/metabase/scenarios/question/snippets.cy.spec.js

```diff
@@ -1,59 +1,96 @@
 import { signInAsNormalUser, restore, modal } from "__support__/cypress";
 
+function _clearAndRecursivelyTypeUsingLabel(label, string) {
+  // Cypress throws an error when trying to type an empty string
+  if (!string) {
+    return;
+  }
+
+  cy.findByLabelText(label).clear();
+
+  for (const char of string) {
+    cy.findByLabelText(label).type(char);
+  }
+}
+
 describe("scenarios > question > snippets", () => {
   before(restore);
   beforeEach(signInAsNormalUser);
 
   it("should let you create and use a snippet", () => {
+    const snippet = {
+      name: "stuff-snippet",
+      sql: "stuff",
+    };
+    // Single quotes are easy to miss here, but are very important!
+    // We're not using SQL keywords or actual tables, so our string must be escaped.
+    const escapedSQL = `'${snippet.sql}'`;
```

Comment:
sorry, another nitpick:
This is creating a string literal, but it isn't really "escaped". If `snippet.sql` were `"stu'ff"`, it wouldn't produce `"'stuff\'ff'"`. It doesn't need to be escaped, I was just calling out the misnomer.

The specific sql content isn't the point of the test here. I used a string literal to keep it simple. If `"'stuff'"` is weird with its nested quotes, I think you could change it to `1 + 1` or something.

Reply:
`1 + 1` might be better actually because you can check that it's evaluated to `2` in the result.

Reply:
It's not a nitpick - you're helping a lot with your comments 🙏
I'll definitely get rid of this and try with `1+1` for the second case.